modules/Bio/EnsEMBL/Variation/VariationFeature.pm

```diff
@@ -1934,7 +1961,14 @@ sub hgvs_genomic {
     ### Apply HGVS 3' shift if required
     my $offset = 0;
     my $lookup_order = 1;
-    my $var_class  =  $self->var_class();
+
+    if ($is_multi_allelic) {
+      # fix for multi-allelic variants
+      ($ref_allele, $allele, $chr_start, $chr_end, my $change) = @{trim_sequences($ref_allele, $allele, $chr_start)};
+      $ref_start += $change;
```

Comment:
`$change` is a boolean value, so it may not capture how many bases we want `$ref_start` shifted by. 
Are we interested in 
```
my $old_chr_start = $chr_start;
($ref_allele, $allele, $chr_start, $chr_end, my $change) = @{trim_sequences($ref_allele, $allele, $chr_start)};
my $offset = $chr_start - $old_chr_start;
$ref_start += $offset;
```

Example to test: `chr8    143694490   143694492   GGG/GG/G    +   var_test`